README.rst

```diff
@@ -4,19 +4,18 @@ NuoDB Client Package
 
 .. contents::
 
-This project bundles the latest publicly-available versions of NuoDB_
-database clients into a single downloadable file.  The official versions of
-the NuoDB Client Package file are available both from `GitHub Releases`_ as
-well as from the `NuoDB Client Portal`_.
+The NuoDB Client Package bundles the latest publicly-available NuoDB_ database client verions into a single downloadable
+file. The official versions of the NuoDB Client Package file are available from the NuoDB `GitHub Releases`_ page and from the
+NuoDB `Community Edition`_ webpage.
```

Comment:
These lines are still too long.  Please restrict line lengths to about 75 chars per line or so.  These are running >120 chars per line.